packages/region/integration-tests/__tests__/region-module.spec.ts

```diff
@@ -63,7 +63,6 @@ describe("Region Module Service", () => {
       expect.objectContaining({
         id: createdRegion.id,
         name: "Europe",
-        currency_code: "EUR",
```

Comment:
question: wouldn't this field still be on the data model? That would in any case be ideal.

Reply:
Should be good now.